src/Model/GroundWaterFlow/gwf3uzf8.f90

```diff
@@ -2219,6 +2220,13 @@ subroutine read_cell_properties(this)
             'SURFDEP for uzf cell', i,                                           &
             'must be greater than 0 (specified value is', surfdep, ').'
           call store_error(errmsg)
+        endif
+        if(surfdep >= this%GWFTOP(i) - this%GWFBOT(i)) then
+          write(errmsg,'(a,1x,i0,1x,a)')                                        &
+            'SURFDEP for uzf cell', i,                                           &
+            'cannot be greater than the cell thickness.'
+          call store_error(errmsg)
+          surfdep = 0.2 * (this%GWFTOP(i) - this%GWFBOT(i))
```

Comment:
Is this statement still needed?

Reply:
Have removed the line of code that modified surfdep